test/test_torch.py

```diff
@@ -6691,15 +6691,19 @@ def test_inverse(self, device):
         expected_inv = torch.as_tensor(inv(matrices.cpu().numpy()))
         self.assertEqual(matrices_inverse, expected_inv.to(device))
 
+    @unittest.skipIf(not TEST_NUMPY, 'NumPy not found')
     @onlyOnCPUAndCUDA
     @dtypes(torch.int8, torch.int16, torch.int32, torch.int64)
     def test_signed_shift(self, device, dtype):
         "Ensure that signed integer bit shifting works as expected."
-        a = torch.tensor([-10, 10], device=device, dtype=dtype)  # [11...1110110, 1010]
+        values = [-10, 10]
+        a = torch.tensor(values, device=device, dtype=dtype)  # [11...1110110, 1010]
         expected_l = torch.tensor([-40, 40], device=device, dtype=dtype)  # [11...11011000, 101000]
         self.assertEqual(a << 2, expected_l)
+        self.compare_with_numpy(lambda x: x << 2, lambda x: np.left_shift(x, 2), values, device, dtype)
```

Comment:
you can send `a` directly as `tensor_like` argument to `compare_with_numpy`.

Reply:
Fixed. I forgot that I can remove device and dtype in kwargs.